### ISSUE TITLE
elpy-profile with ipython fails to run

### ISSUE THREAD

**Issue by MRFaria** [jorgenschaefer/elpy#1114]
Running 
`M-x: elpy-profile-buffer-or-region `

fails when the emacs python interpreter is set to ipython. 

I had a brief look in elpy-profile.el and it looks the issue arises because ipython doesn't have the "-o" command line option to write the python output to a file.
`(proc-cmd (list python-shell-interpreter "-m" "cProfile" "-o" prof-file file))`

Here is my elpy config:

```
(setq python-shell-interpreter "ipython")
(setq python-shell-interpreter-args "-i")
(elpy-enable)
(setq elpy-rpc-backend "jedi")
(elpy-use-ipython "ipython")
```
My elpy version 1-14-1
edit: I am running ipython version 4.2 because of other issues I had with the newer versions

**Comment by jorgenschaefer** [comment]
Hello, and thanks for the report! I suspect elpy-profile.el should use its own variable for the command …

**Comment by memeplex** [comment]
Same problem here. It's not difficult to workaround using an advice, though.

**Comment by galaunay** [comment]
@jorgenschaefer Do you think it would be a possible to use `elpy-rpc-python-command` instead of having a new option to set ?

**Comment by jorgenschaefer** [comment]
Yeah, that looks sensible considering it uses the `-m`argument.

**Comment by galaunay** [comment]
Should be fixed by #1528.
I am closing this.

Please re-open if the issue is still happening.
